### ISSUE TITLE
Add stereotypes from source hází výjimky do konzole

### ISSUE THREAD

**Issue by ledsoft** [opendata-mvcr/ontoGrapher#4]
Při pokusu o přidání stereotypů ze vzdálené URL dojde k chybě. Konzole prohlížeče hlásí:

```
MenuSettingsNodes.js:137 Uncaught TypeError: Cannot read property 'state' of undefined
    at handleLoadStereotypes (MenuSettingsNodes.js:137)
    at HTMLUnknownElement.callCallback (react-dom.development.js:347)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:397)
    at invokeGuardedCallback (react-dom.development.js:454)
    at invokeGuardedCallbackAndCatchFirstError (react-dom.development.js:468)
    at executeDispatch (react-dom.development.js:600)
    at executeDispatchesInOrder (react-dom.development.js:619)
    at executeDispatchesAndRelease (react-dom.development.js:725)
    at executeDispatchesAndReleaseTopLevel (react-dom.development.js:733)
    at forEachAccumulated (react-dom.development.js:707)
```

**Comment by bindetad** [comment]
O jakou URL se jedná? Myslím, že je bug opraven v 544c4ea605c7f56fc5bbbba42b683a20a53da11e, nicméně bych si to rád otestoval i s tvojimi vstupními daty.

**Comment by ledsoft** [comment]
Tuším, že jsem zkoušel https://raw.githubusercontent.com/opendata-mvcr/ssp/master/z-sgov/pln%C3%BD/z-sgov-model.ttl, ale řekl bych, že tahle chyba by na vstupních datech záviset nemusela...

**Comment by martinnec** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Práce na issues v ontoGrapher #1](https://trello.com/c/RCIYuzZP/22-pr%C3%A1ce-na-issues-v-ontographer-1)

**Comment by bindetad** [comment]
Bug je opraven v b693b610b8797c4d349f5b25b63a3a8d040eb8b8, uzavírám.

**Comment by blcham** [comment]
Reopening the bug -- using same url as above and putting it into "Using source" tab of  "Relationship overview" or "Classes overview" produces same error output in javascript console:
```
backend.js:6 TypeError: Cannot read property 'on' of null
    at s._import (bundle.js:110)
    at bundle.js:110
    at new Promise (<anonymous>)
    at s.import (bundle.js:110)
    at bundle.js:211
```

**Comment by blcham** [comment]
Moreover unification of "Class settings" and "Relationship overview" would be nice.

**Comment by bindetad** [comment]
I have changed the name of the labels (if you meant the dialogue headers) as well as (hopefully) fixed the bug in 5343096fdbafaeebbeb9a4e34a3e6bde80ee1626. Please check.

**Comment by blcham** [comment]
Both "Class settings" and "Relationship settings" loads data now however operation "Add classes" is not idempotent -- the list of "classes" does not behave as a set, while the list of "relationships"  behaves as a set.
It means that if I hit "Add classes" multiple times It adds the same "class" multiple times there. Unless you fix it here I will create a separate issue for that.
